Formula/php-cs-fixer.rb

```diff
@@ -1,15 +1,15 @@
 class PhpCsFixer < Formula
   desc "Tool to automatically fix PHP coding standards issues"
   homepage "https://cs.symfony.com/"
-  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.18.6/php-cs-fixer.phar"
-  sha256 "f74b385d358aa2d81b4e2f8cc7ede697b36a0d83b7c62231cb653fa98c6d4411"
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v3.0.0/php-cs-fixer.phar"
+  sha256 "993f3d300db32c4158891bc8a13a7f91f7dac85ce4eda209b3582f0e329b3990"
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "d1336616dc3319ea53d1a6e299b879fbee7df68c0bbe8ce4c14a01ba33b53791"
   end
 
-  depends_on "php" if MacOS.version <= :el_capitan
+  depends_on "php" if MacOS.version <= :high_sierra
```

Comment:
```suggestion
  uses_from_macos "php", since: :catalina
```

Reply:
PHP CS Fixer 3.0.0 requires PHP 7.1 or newer. IIRC, Mojave bundled PHP 7.1 already.

Reply:
Oops, definitely meant to type Mojave